examples/noop_malloc.cpp

```diff
@@ -0,0 +1,18 @@
+//////////////////////////////////////////////////////////////////////////////
```

Comment:
I like the idea of keeping the example.  But I would call it noop_allocator.cpp instead of noop_malloc.cpp.